tests/Microsoft.Identity.Test.Performance/Microsoft.Identity.Test.Performance.csproj

```diff
@@ -8,7 +8,7 @@
     <!-- https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/code-generation#debugtype -->
     <!--<DebugType>pdbonly</DebugType>
     <DebugSymbols>true</DebugSymbols>-->
-    <LangVersion>latest</LangVersion>
+    <LangVersion>12</LangVersion>
```

Comment:
Does this need to be specified here? We should use a Directory.Builds.props for common stuff.

Reply:
Indeed, but seems like changing this stuff is finicky, so I'll leave it out of this PR.